src/main/kotlin/nl/info/zac/app/zaak/ZaakKoppelenRestService.kt

```diff
@@ -115,7 +116,12 @@ class ZaakKoppelenRestService @Inject constructor(
         (areBothOpen(sourceZaak, targetZaak) || areBothClosed(sourceZaak, targetZaak)) &&
             sourceZaak.hasLinkRights() &&
             targetZaak.hasLinkRights() &&
-            targetZaak.isLinkableTo(sourceZaak, relationType)
+            targetZaak.isLinkableTo(sourceZaak, relationType) &&
+            isAlreadyLinked(sourceZaak) &&
```

Comment:
This is not correct right? Because you can link one hoofdzaak to multiple deelzaken. So if you have an existing hoofdzaak we should allow linking another deelzaak to it. It seems with this change we no longer allow this? That is not good. We should also have a test for this scenario.